api/nodes/serializers.py

```diff
@@ -313,6 +314,13 @@ def update(self, instance, validated_data):
         pass
 
 
+class NodeLinksDetailSerializer(NodeLinksSerializer):
+    """
+    Overrides NodeLinksSerializer to make id required.
+    """
+    id = IDField(source='_id', required=True)
```

Comment:
Can you clarify why id is no longer read-only? (even though it is in the parent)